source/uwp/Renderer/dll/CppWinRTIncludes.h

```diff
@@ -32,6 +34,8 @@ namespace winrt
 	using namespace ::winrt::Windows::Foundation;
```

Comment:
nit: could you please fix the spacing here?:)
@paulcam206 @licanhua I can't remember, have we added automatic formatting for `UWP Renderer` code?

Reply:
for some reason it looks fine now, my bad :)